client/src/main/java/com/aliyun/emr/rss/client/compress/RssZstdDecompressor.java

```diff
@@ -39,10 +39,9 @@ public int getOriginalLen(byte[] src) {
 
   @Override
   public int decompress(byte[] src, byte[] dst, int dstOff) {
-    int token = src[MAGIC_LENGTH] & 0xFF;
-    int compressionMethod = token & 0xF0;
+    int compressionMethod = src[MAGIC_LENGTH] & 0xFF;
```

Comment:
Do we want keep compatibality at the current stage? If yes, it should be
```suggestion
    int compressionMethod = src[MAGIC_LENGTH] & 0xF0;
```

Reply:
May I know some example about the compatibility?

Reply:
Emm, leave it, the compression/decompression only occurs on the client side, and won't be shared across applications, so the change is safe.